mlflow/utils/credentials.py

```diff
@@ -90,10 +117,11 @@ def _check_databricks_auth():
         w = WorkspaceClient()
         # If credentials are invalid, `clusters.list()` will throw an error.
         w.clusters.list()
-        _logger.info(
-            "Succesfully signed in Databricks! Please run `mlflow.set_tracking_uri('databricks')` "
-            "to connect MLflow to Databricks tracking server."
-        )
+        _logger.info("Succesfully signed in Databricks!")
+        # Connect MLflow to Databricks tracking server.
+        from mlflow import set_tracking_uri
+
+        set_tracking_uri("databricks")
```

Comment:
Feel free to split this into a separate smaller PR so we can merge :)

Do we need to make any similar changes for `mlflow.login()` with OSS MLflow auth?

Reply:
It's harder on that side, in Databricks the tracking uri is always "databricks", for OSS it's a customizable URL.